config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics.yaml

```diff
@@ -0,0 +1,30 @@
+periodics:
+- name: periodic-cluster-api-provider-azure-conformance-v1alpha3
+  decorate: true
+  decoration_config:
+    timeout: 3h
+  interval: 12h
```

Comment:
We're probably going to want to run this more frequently to get signal.
The capa one runs every 6h. Let's try somewhere between 2-4h to start.

Reply:
I am going to leave this at every 12 hours for now as Azure is under capacity constraints because of COVID effort, will increase the frequency later. Source: https://azure.microsoft.com/en-us/blog/update-2-on-microsoft-cloud-services-continuity/

Reply:
SGTM. Thanks for the context.